src/Settings.js

```diff
@@ -122,6 +128,11 @@ export class Settings {
             if (this.globalConfig.isTopFrame) {
                 return Settings.defaults.availableInputTypes
             }
+
+            if (this._availableInputTypes) {
+                return this.availableInputTypes
```

Comment:
This basically avoids reaching out to the device again if the data is already available. Prepares for the upcoming migration of other platforms to include `availableInputTypes` with the runtime config.